charts/incubator/hyperswitch-sdk/templates/sdk-demo-app/deployment.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
   generation: 1
-  name: {{ .Values.sdkDemo.labels.app }}-0f746cb72517
```

Comment:
instead of writing this everywhere, can you add this into the template file and mention it everywhere, this would reduce the surface area of error

Reply:
yes, let us do all the refactoring in a separate PR.